iroha2/data_model/asset/__init__.py

```diff
@@ -13,8 +13,10 @@
 )
 from ..domain import Id as DomainId
 from ..isi import Registrable
+from .. import wrapper
 
 
+@wrapper(_DefinitionId)
```

Comment:
These need to be explained in-code. Why are we wrapping this structure in what seems to be the same structure with a reserved pattern `_[::capital::]`

Reply:
Documented `wrapper`